### PR TITLE
Recommended Indexes

### PR DESCRIPTION
**Pull request by mohitkhullar**

### CODE REVIEW

**Comment by akshatsikarwar** [comment]
Plz also wait for #459 
I promise will get it in today.

**Comment by ldennis** [comment]
Is there any comdb2 modification to sqlite source? If so, can you highlight them?

**Comment by mohitkhullar** [comment]
Added comments on comdb2 changes where they were not present.

**Comment by ldennis** [comment]
Thanks. I will take a look

File: db/sqlinterfaces.c [hunk 1]
```diff
@@ -4030,6 +4031,67 @@ static int get_prepared_stmt_int(struct sqlthdstate *thd,
         return handle_bad_transaction_mode(thd, clnt);
     }
 
+    if (clnt->is_expert) {
```
Comment (ldennis) [inline db/sqlinterfaces.c]: Move this into a function and put it into `handle_non_sqlite_requests`, like the way we did for explain?

File: db/sqlinterfaces.c [hunk 2]
```diff
@@ -4030,6 +4031,67 @@ static int get_prepared_stmt_int(struct sqlthdstate *thd,
...
+#if 0
+          for(int i=0; i<nQuery; i++){
+            const char *zSql = sqlite3_expert_report(p, i, EXPERT_REPORT_SQL);
+            const char *zIdx = sqlite3_expert_report(p, i, EXPERT_REPORT_INDEXES);
+            const char *zEQP = sqlite3_expert_report(p, i, EXPERT_REPORT_PLAN);
+            if( zIdx==0 ) zIdx = "(no new indexes)\n";
+            fprintf(stdout, "-- Query %d ----------------------------------\n",i+1);
+            fprintf(stdout, "%s\n\n", zSql);
+            fprintf(stdout, "%s\n%s\n", zIdx, zEQP);
+          }
```
Comment (ldennis) [inline db/sqlinterfaces.c]: If we are not supporting multiple queries, lets remove this?

**Comment by ldennis** [comment]
Reviewed with @mohitkhullar in person. Looks good to me.

**Comment by mohitkhullar** [comment]
Thanks Lingzhi

**Pull request merged**